front/app/components/admin/SlugInputField/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
The slug input + some other stuff (errors) that we're using the project/folder forms.